joomla/joomla-cms
# [4.0] fix ugly bug in grunt uglify

Pull Request for Issue # .

### Summary of Changes
- Delete all minified javascript files before running uglify 
- rename jQuery.simplecolors.js to simplecolors.js (multiple . produce wrong minified files :( )


### Testing Instructions

Just review and run grunt scripts  to verify that all files are still created

@wilsonge plz merge asap

### Expected result



### Actual result



### Documentation Changes Required
